WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Customs/ShippingLabelCustomsFormInput.swift

```diff
@@ -66,20 +119,35 @@ private extension ShippingLabelCustomsFormInput {
                                                          comment: "Title for the return policy in Customs screen of Shipping Label flow")
         static let contentTypeTitle = NSLocalizedString("Content Type",
                                                         comment: "Title for the Content Type row in Customs screen of Shipping Label flow")
+        static let contentExplanationTitle = NSLocalizedString("Content Details",
+                                                               comment: "Title for the Content Details row in Customs screen of Shipping Label flow")
+        static let contentExplanationPlaceholder = NSLocalizedString("Type of contents",
+                                                                     comment: "Placeholder for the Content Details row " +
+                                                                        "in Customs screen of Shipping Label flow")
         static let restrictionTypeTitle = NSLocalizedString("Restriction Type",
                                                             comment: "Title for the Restriction Type row in Customs screen of Shipping Label flow")
+        static let restrictionCommentTitle = NSLocalizedString("Restriction Details",
+                                                               comment: "Title for the Restriction Details row in Customs screen of Shipping Label flow")
+        static let restrictionCommentPlaceholder = NSLocalizedString("Type of restriction",
+                                                                     comment: "Placeholder for the Restriction Details row in " +
+                                                                        "Customs screen of Shipping Label flow")
         static let itnTitle = NSLocalizedString("ITN",
                                                 comment: "Title for the ITN row in Customs screen of Shipping Label flow")
         static let itnPlaceholder = NSLocalizedString("Enter ITN (optional)",
```

Comment:
nit: Sorry for missing this before — I noticed that "optional" is capitalized elsewhere in the form and in the designs, so it should be capitalized here too:

```suggestion
        static let itnPlaceholder = NSLocalizedString("Enter ITN (Optional)",
```

Reply:
This has been fixed in ba43182c91c25c634baa84b13241161660a2ab2d.